src/routes/edit/[id]/+page.svelte

```diff
@@ -114,7 +114,7 @@
 		<button type="submit" class="btn btn-outline mr-5" name="draft" value="true">
 			Entwurf speichern
 		</button>
-		<button type="submit" class="btn btn-primary">Hochladen</button>
+		<button type="submit" class="btn btn-primary">Veröffentlichen</button>
```

Comment:
Hä, bei mir steht da immernoch hochladen

Reply:
Check mal bitte ob in der url `/edit` oder `/create` steht. auf `/edit` sollte Veröffentlichen stehen

Reply:
Doch ja bei edit steht veröffentlichen :)